Thorium-Sim/thorium-nova
# Navigation search

## Description

Add a search field to the navigation card.

## Related Issue








Closes #388 

## How do you know the changes work correctly?






Manual test
- Start a flight
- Go to the navigation card
- Type "Alpha" into the search field
- Choose one of the options
- The camera should center on that object.


## Types of changes



      change)

## Checklist:
